playbooks/devel.yml

```diff
@@ -1,21 +1,28 @@
 ---
-- hosts: devel
+- hosts: all
   become: true
   vars:
     katello_repositories_version: nightly
-    foreman_installer_scenario: katello-devel
-    foreman_installer_options_internal_use_only:
-      - "--disable-system-checks"
-      - "--katello-devel-github-username {{ katello_devel_github_username }}"
-      - "--katello-devel-enable-ostree=true"
+    katello_repositories_environment: staging
+    foreman_installer_skip_installer: true
     foreman_installer_additional_packages:
-      - foreman-installer-katello-devel
+      - foreman-installer-katello
   roles:
     - etc_hosts
     - epel_repositories
     - ostree_repositories
     - puppet_repositories
     - foreman_repositories
     - katello_repositories
-    - foreman_devel
```

Comment:
Is there a reason you're removing this? It includes some SSH config.

Reply:
Added this back further down.

Reply:
I don't see it.